### PR TITLE
Timer Reset Warning (#9)

### PR DESCRIPTION
**Pull request by Burniee**
==Changelog==
- Added warning message to setting popup

==Testing==
- Jest and Cypress tests pass

==Version Compatibility==
- Full Compatibility, since nothing crazy was changed in the code to warrant any compatibility issues

==Detail Description==
- Updated setting popup menu to reflect the fact that any changes to settings will cause a reset in timer

### CODE REVIEW

**Comment by Burniee** [comment]
Linting fail is from code did I did not work on

**Comment by Burniee** [comment]
d3069df8eec641efc9815084165656d5

**Comment by kat066** [comment]
Lgtm, but can you wait until I merged mine? I am finishing up my refactoring, and this might require some change.

**Comment by Burniee** [comment]
Yeah, that's fine, that's what I initially wanted to do, but it seemed you were taking a while. But yeah, that's chill

**Review by zianlin — DISMISSED** [review]
Changes look good, I don’t see anything wrong 👍

**Comment by kat066** [comment]
Thanks.

**Comment by broiyn** [comment]
also dont forget to fix the linting failure before merging, looks like there's just some extra whitespace

**Comment by Burniee** [comment]
Yeah, it's already been fixed, need to wait for Kai's changes to be pushed

**Comment by kat066** [comment]
Actual changes lgtm. But your commit also deletes the video and modifies package.json.

File: .gitmessage.txt
Comment (kat066) [inline .gitmessage.txt:1]: This is a commit message template, not the actual commit message, please do not modify. Simply run `git config --global commit.template .gitmessage.txt` to activate the template, and when you do git commit, this will show up, then you can modify there.

**Review by kat066 — CHANGES_REQUESTED** [review]
Actual changes lgtm. But your commit also deletes the video and modifies package.json and commit message template.

**Pull request closed without merging**
